simpleTest1/appFirstPb.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function (array) {
+    
+    var sum = 0;
+    let notNumber = 0;
+
+    for (var index = 0; index < array.length; index++) {
+        if (parseInt(array[index]) === array[index]) {
```

Comment:
The problem said that I have to do the sum of even numbers. I know that even/odd only refers to integers, so that is why I verified If I had only integers. An array can have different types of items, of string type, boolean type etc. On best-case scenario I think I can assume I have only integers.